sql/updates/world/3.3.5/9999_99_99_99_world_quest_get_me_outa_here.sql

```diff
@@ -0,0 +1,32 @@
+--
+SET @GUID := xxx;
+DELETE FROM `gameobject` WHERE `guid`=@GUID;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(@GUID, 182531, 571, 0, 0, 1, 1, 4413.98, 5368.06, -15.5602, 3.92534, -0, -0, -0.924195, 0.381922, 300, 100, 1, "", 0);
+UPDATE `creature` SET `position_x`=4414.223, `position_y`=5367.299, `position_z`=-15.49403, `orientation`=0.3665192, `spawntimesecs`=300 WHERE `id`=25589;
```

Comment:
Recommendation: add a blank line for readability before
- `DELETE FROM`
- `UPDATE creature`

Reply:
Done!